index.html

```diff
@@ -726,15 +745,14 @@
     </paper-header-panel>
   </neon-animated-pages>
   <paper-dialog id="dialog">
-    <div id="settingspanel"><h2>Settings</h2>
-    <h3>Privacy settings</h3>
-    <div class="inline"><span class="leftfloat">Show other EPS students my photo</span><paper-toggle-button checked class="rightfloat" id="sharephototoggle"></paper-toggle-button></div>
-    <div class="inline"><span class="leftfloat">Show other EPS students my schedule</span><paper-toggle-button checked class="rightfloat" id="sharescheduletoggle"></paper-toggle-button></div>
-    <paper-button onclick="submitUpdatePrivacy()" raised class="settingsbutton" id="updatesettings">UPDATE SETTINGS</paper-button>
-    <h3>Change password</h3>
-    <paper-input label="Old password" id="oldpassword" type="password"></paper-input>
-    <paper-input label="New password" id="newpassword" type="password"></paper-input>
-    <paper-button onclick="submitChangePassword()" raised class="settingsbutton" id="changepassword">CHANGE PASSWORD</paper-button></div>
+    <div id="settingspanel">
+      <h2 class="settingstitle">Settings</h2>
+      <div class="settingsdividerline"></div>
```

Comment:
The hr tag has not been used for horizontal lines since HTML 4.01, [view the w3schools documentation for more details](http://www.w3schools.com/tags/tag_hr.asp)